novatel_oem7_driver: 1.0.0-1 in 'melodic/distribution.yaml' [bloom]

novatel-applications-engineering opened the pull request:
Increasing version of package(s) in repository `novatel_oem7_driver` to `1.0.0-1`:

- upstream repository: https://github.com/novatel/novatel_oem7_driver.git
- release repository: https://github.com/novatel-gbp/novatel_oem7_driver-release.git
- distro file: `melodic/distribution.yaml`
- bloom version: `0.9.3`
- previous version for package: `null`

novatel-applications-engineering committed:
novatel_oem7_driver: 1.0.0-1 in 'melodic/distribution.yaml' [bloom]

clalancette commented on melodic/distribution.yaml:
```diff
@@ -6139,6 +6139,18 @@ repositories:
   novatel_oem7_driver:
+    doc:
+      type: git
+      url: https://github.com/novatel-gbp/novatel_oem7_driver-release.git
```
The doc block should not point to the release repository; instead, it should point to the source repository and branch (similar to the `source` section below).

novatel-applications-engineering replied:
@clalancette, thank you very much. This change has been made and is now included in db1fc4e.

novatel-applications-engineering committed:
Update distribution.yaml

Revised novate_oem7_driver's doc url to point to our source repo and master branch, as has been recommended.

novatel-applications-engineering reviewed (commented):
Thank you, we've made these changes now.
